cdci_data_analysis/plugins/dummy_instrument/data_server_dispatcher.py

```diff
@@ -17,9 +17,7 @@
 
 from __future__ import absolute_import, division, print_function
 
-import urllib
-from builtins import (bytes, str, open, super, range,
-                      zip, round, input, int, pow, object, map, zip)
+import os
```

Comment:
If dropping this part of 2-to-3 transition, why not also drop future imports?

Reply:
We can just do it all at once in a single commit, once enough tests are implemented.
Similarly, we can make a single commit for style alignment.
Commits mix features and style and larger then necessary.